yw/Software/test/Checker1Test.java

```diff
@@ -0,0 +1,138 @@
+import static org.junit.Assert.*;
```

Comment:
Should we delete libraries that are not used?

Reply:
All libraries that are not used have been deleted